docs/data_managers/using_safedata/publishing_datasets.md

```diff
@@ -0,0 +1,90 @@
+# Publishing a dataset
+
+<!-- markdownlint-disable MD046 MD033 -->
+
+This page provide examples of using the `safedata_validator` package to publish
+validated datasets. The examples assume that a user has provided a SAFE formatted
+dataset and an additional ZIP file containing additional files:
+
+* `Example.xlsx`
+* `Supplementary_files.zip`
+
+Both of the use cases below include the creation and includsion of a GEMINI compliant
+XML metadata file in a published dataset. We recommend this as good practice, but it is
+optional.
+
+## Validating and publishing as a new dataset
+
+The [`safedata_zenodo publish_dataset`
+command](../command_line_tools/safedata_zenodo.md#the-publish_dataset-subcommand) is
+the main function for publishing a dataset. The example below shows it being used to
+publish a dataset and additional external files:
+
+```sh
+safedata_zenodo publish_dataset Example.xlsx -e  Supplementary_files.zip
+```
+
+That command packages up a set of operations needed to publish a dataset. The tabs
+examples below show the underlying workflow, either using the command line interface or
+working from within Python,  for publishing these data and accompanying metadata as a
+completely new dataset using `safedata_validator`. You would typically not need to use
+these individual commands: this information is here to show what is going on under the
+hood.
+
+=== "bash"
+
+    ```sh
+    {%
+    include "data_managers/using_safedata/publish_script.sh"
+    %}
+    ```
+
+=== "python"
+
+    ```python
+    {%
+    include "data_managers/using_safedata/publish_script.py"
+    %}
+    ```
+
+## Validating and publishing as an update
+
+Zenodo can hold multiple versions of a dataset, allowing you to publish updates and
+corrections. When you create a new dataset, that dataset is assigned a record id but
+is also automatically also given a **concept record id**. These are often sequential:
+in the previous example, the specific record ID for the dataset was `1143714` - the
+concept record ID might be `1143713`. The concept record id is used to group versions of
+a dataset and the associated DOI is used to redirect to the most recent version of the
+dataset.
+
+Releasing a new version generates a copy of the most recent version of a dataset,
```

Comment:
When you say "generates a copy" do you mean locally or does it upload it to Zenodo?

Reply:
It's created on Zenodo. Clarified